docs/02-providers/aws/events/01-apigateway.md

```diff
@@ -187,6 +187,36 @@ module.exports.hello = (event, context, cb) => {
 }
 ```
 
+#### Custom status codes
+
+You can override the detauls status codes supplied by serverless if you need to change the default code, add/remove codes, or change the templates and selection process that dictates what code is returned.
+
+If you specify a status code with a pattern of '' that will become the default response code. See below on how to change the default to 201 for post requests.
+
+If you omit any default status code. A standard default 200 status code will be generated for you.
+
+```yml
+functions:
+  create:
+    handler: posts.create
```

Comment:
imo we should probably keep all our example handlers as `hander.something` ... since `handler.js` is the default name when creating a service. This should make things a bit easier for beginners at least